playground/ClusterStress/OnlineReqGen.cs

```diff
@@ -75,10 +75,10 @@ public byte[] GenerateKeyBytes(out int slot)
             int key = randomGen ? (zipf ? zipfg.Next() : keyRandomGen.Next(DbSize)) : (keyIndex++ % DbSize);
             slot = Garnet.common.NumUtils.HashSlot(Encoding.ASCII.GetBytes(key.ToString()));
             byte[] keyBytes = GetClusterKeyBytes(key);
-#if DEBUG
```

Comment:
It might be best to use the full namespace for Debug instead of removing the guard. #ifdef is there to avoid the cost of NumUtils.HashSlot call.